tests/passing-configs/index.js

```diff
@@ -14,8 +15,6 @@ export function filePathToConfigObj(filepath) {
   return {
     config,
     filepath,
-    name: path.parse(filepath).name,
+    name: pathParse(filepath).name
```

Comment:
No idea, maybe @miguelc1221 used `--no-verify`. No worries. Just glad to see this PR :D